packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -393,6 +400,34 @@ export const HelpCenterContactForm = () => {
 		}
 	};
 
+	const getSupportedLanguages = ( supportType: string, locale: string ) => {
+		const isLiveChatLanguageSupported = (
+			config( 'livechat_support_locales' ) as Array< string >
+		 ).includes( locale );
+
+		const isLanguageSupported = ( config( 'upwork_support_locales' ) as Array< string > ).includes(
+			locale
+		);
+
+		switch ( supportType ) {
+			case SUPPORT_HAPPYCHAT:
+				return ! isLiveChatLanguageSupported;
+
+			case SUPPORT_TICKET:
+			case SUPPORT_CHAT_OVERFLOW:
+			case SUPPORT_UPWORK_TICKET:
+				return ! isLanguageSupported && ! [ 'en', 'en-gb' ].includes( locale );
```

Comment:
If we are going to be using the config for the languages, it could be helpful to use the english one as well `english_locales`. But this is just a nitpick haha.